analytics/partner.js

```diff
@@ -8,4 +8,16 @@
   analyticsHooks.on('partner:non-claimed', function (options) {
     analytics.track('Non-claimed partner page', _.extend(options, {nonInteraction: 1}))
   })
+
+  $('#partner-contact .email-gallery').click(function () {
+    analytics.track('Clicked Contact Gallery Via Email', {
+      gallery_id: sd.CURRENT_PATH.split('/')[1] // gallery id
```

Comment:
I could add a data attribute [here](https://github.com/artsy/force/blob/master/apps/partner2/templates/contact_info.jade#L4) and stick the gallery id in it then pull it off the clicked element. Do you prefer that?